### DISCUSSION TITLE
关于混合搜索方面咨询

### DISCUSSION THREAD

**Discussion by zhenglongkang228** [milvus-io/milvus#17049] [Q&A and General discussion]
请问milvus有为每个字段设置索引吗？混合搜索时是怎么和faiss的索引结合的？

**Comment by yhmo** [comment]
primary key字段有哈希索引(unordered_map)，其他标量字段没有索引。
混合搜索的通俗讲解参考这篇：https://zhuanlan.zhihu.com/p/447703140
